### PR TITLE
feat: (builder) add config

### PR DESCRIPTION
**Pull request by pbosio**
adds configuration class for builder

# What? 
add and handle configuration sent from builder

# Why? 
we need to be able to tweak some values and behaviors dynamically

### CODE REVIEW

File: unity-client/Assets/Builder/Scripts/DCLBuilderBridge.cs [hunk 1]
```diff
@@ -295,6 +295,24 @@ public void GetCameraTargetBuilder(string id)
             }
         }
 
+        public void SetBuilderConfiguration(string payloadJson)
+        {
+            if (LOG_MESSAGES) Debug.Log($"RECEIVE: SetBuilderConfiguration {payloadJson}");
+            DCLBuilderConfig.SetConfig(payloadJson);
+
+            if (currentScene)
```
Comment (BrianAmadori) [inline unity-client/Assets/Builder/Scripts/DCLBuilderBridge.cs]:
```suggestion
            if (!currentScene)
                return;
```

**Review by BrianAmadori — APPROVED** [review]
LGTM!

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs [hunk 2]
```diff
@@ -163,46 +166,18 @@ public virtual void SetUpdateData(LoadParcelScenesMessage.UnityParcelScene data)
...
-                        matTransition.useHologram = false;
-                        matTransition.fadeThickness = 20;
-                        matTransition.OnDidFinishLoading(finalMaterial);
-                    }
-                    else
-                    {
-                        plane.GetComponent<MeshRenderer>().sharedMaterial =
-                            Utils.EnsureResourcesMaterial("Materials/DefaultPlane");
-                    }
-                }
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs:205]: Ty for this ❤️

File: unity-client/Assets/Builder/Scripts/DCLBuilderBridge.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Builder/Scripts/DCLBuilderBridge.cs]: done 🙃

**Review by sandrade-dcl — APPROVED** [review]

**Pull request merged**